docker/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:latest
+
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install -y nodejs 
+
+RUN mkdir -p /usr/src/app
+
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app
+
+RUN npm install
```

Comment:
Do `npm install --only=prod` for the production version of the app, or better,  you can make two different files, one for building a development image, and one for production.